services/storage/src/simcore_service_storage/api/v0/openapi.yaml

```diff
@@ -1296,6 +1296,10 @@ components:
           type: object
           title: Quality
           description: Object containing Quality Assessment related data
+        sharingNetworks:
```

Comment:
This is a side effect of what I spoke above about the "monolitic model" with the project. I think we all agree that it has little use this information in the storage's model of the project.

Reply:
The vied will be moved to a separate table. Will also disappear from here.